.env.example

```diff
@@ -17,4 +17,4 @@ SB_BIG_THRESHOLD=24
 <<<<<<< HEAD
 SB_RATELIMIT=30
 =======
->>>>>>> 6ce7152bcf2777976527b540ea014b9967d772fa
+>>>>>>> 6ce7152bcf2777976527b540ea014b9967d772fa
```

Comment:
yeah that's on me, git blame points at #75.

If you could just remove the merge artifacts altogether, that would be big

Reply:
Removed.